tests/performance_tests.py

```diff
@@ -0,0 +1,47 @@
+from time import sleep
+from datetime import datetime
+
+import unittest
```

Comment:
We have here is import block that is in incorrect order, I would improve the order of import